src/languages/twig.js

```diff
@@ -28,17 +29,17 @@ export default function(hljs) {
   var FILTER = {
     begin: /\|[A-Za-z_]+:?/,
     keywords:
-      'abs batch capitalize column convert_encoding date date_modify default ' +
-      'escape filter first format inky_to_html inline_css join json_encode keys last ' +
-      'length lower map markdown merge nl2br number_format raw reduce replace ' +
```

Comment:
Are `markdown` and `filter` removed/deprecated?  Are they still in use in any supported versions?  We try to be conservative in removing keywords if they could potentially still be in use in code snippets, etc.

Reply:
Yes, they were removed.

Reply:
But people are presumable still using Twig 2.0, yes?  The docs are still available.

https://twig.symfony.com/doc/2.x/filters/index.html

Reply:
Then I add it again, I just looked at the 3.0. Thanks :)

Reply:
Wait, I thought `filter` was a tag?

Reply:
It's both :)

Reply:
How should we classify it?

Reply:
Yes, it is. So it's alright.